terraform/production/feature_flags.tf

```diff
@@ -1,8 +0,0 @@
-resource "aws_secretsmanager_secret" "show_historic_data_feature_flag" {
```

Comment:
Just a heads up incase you're not aware, this will have to be removed manually through the AWS console as this was never run. Maybe we delete this file for production but keep for staging as that does work to avoid confusion?

Reply:
We'll also need to remove the environment variable from the serverless config:

https://github.com/LBHackney-IT/social-care-case-viewer-api/blob/e1b935cac6e76ab8c38cf43c0024d47029e526f2/serverless.yml#L38

Reply:
Ah nice, yeah that makes sense!